mne/epochs.py

```diff
@@ -1738,7 +1758,7 @@ def _get_data(
                 is_good, bad_tuple = self._is_good_epoch(epoch, verbose=verbose)
                 if not is_good:
                     assert isinstance(bad_tuple, tuple)
-                    assert all(isinstance(x, str) for x in bad_tuple)
+                    assert all(isinstance(x, (str, tuple)) for x in bad_tuple)
```

Comment:
This part is weird. For any one epoch, the `bad_tuple` should be `tuple of str`, it shouldn't be allowed to be `tuple of tuple of str`.
```suggestion
                    assert all(isinstance(x, str)) for x in bad_tuple)
```
If an epoch is rejected for example because flat check returns `tuple of str` and reject returns `tuple of str` these should be concatenated (with `+` of the two tuples). In theory you should be able to do this with empty tuples, too. I'm surprised it's not already done...?